src/styles/pages/services/services.module.scss

```diff
@@ -0,0 +1,187 @@
+.mainContainer {
+    font-family: Arial; //import later 
+    display: flex;
+    flex-direction: column;
+
+}
+
+.servicesContainer {
+    background-color: #FFFBF0;
+    color: #5B1110;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-bottom: 2rem;
+    padding-top: 2rem;
+    row-gap: 1rem;
+    justify-content: space-between;
+
+    .heading {
+        padding: 1rem;
+        font-size: 1.2rem;
+        word-wrap: break-word;
+    }
+}
+
+.boxesContainer {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    width: 36%;
+    column-gap: 2rem; //do not change gap size 
+
+    .topServiceBox {
+        background-color: #E7D3B8;
+        border-radius: 0.75rem;
+        padding: 2rem;
+        text-align: center;
+        flex-basis: 100%;
+
+
+        >h2 {
+            font-size: 1rem;
+            color: #5B1110;
+            flex-basis: 100%;
+
+        }
+
+        >p {
+            color: black;
+            padding-top: 1rem;
+        }
+
+        >ul {
+            color: black;
+            padding-top: 1rem;
+            columns: 2;
+        }
+    }
+
+    .bottomServiceBox {
+        background-color: #E7D3B8;
+        border-radius: 0.75rem;
+        padding: 2rem;
+        text-align: center;
+        flex-basis: 100%;
+
+        >h2 {
+            size: 200%;
+            color: #5B1110;
+            flex-basis: 100%;
+
+        }
+
+        >p {
+            color: black;
+            padding-top: 1rem;
+        }
+
+        >ul {
+            color: black;
+            padding-top: 1rem;
+            columns: 2;
+        }
+    }
+}
+
+
+// Slider code 
+
+.slideContainer {
+    background-color: #FFFBF0;
+    display: flex;
+    flex-direction: column;
+    color: #D79C1A;
+}
+
+.tabContainer {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-around;
+    background-color: #FFFBF0;
+
+}
+
+.tab {
```

Comment:
slider looks good 👍🏼 , just be sure to add the right font sizes for the tabs (`30px` ~ `1.875rem`)